application/jira/v1/config/tasks.json

```diff
@@ -0,0 +1,1181 @@
+    "instillShortDescription": "Get an issue in Jira",
+    "input": {
+      "description": "Get an issue in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "issue-key"
+      ],
+      "properties": {
+        "issue-key": {
+          "title": "Issue Key",
+          "description": "The key of the issue, e.g. `JRA-1330`",
+          "instillShortDescription": "The key of the issue",
+          "instillUIOrder": 0,
+          "instillFormat": "string",
+          "instillAcceptFormats": [
+            "string"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "string"
+        },
+        "update-history": {
+          "$ref": "#/$defs/common-query-params/update-history",
+          "instillUIOrder": 1
+        }
+      },
+      "required": [
+        "issue-key"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Get an issue in Jira",
+      "instillUIOrder": 0,
+      "$ref": "#/$defs/issue",
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_GET_SPRINT": {
+    "description": "Get a sprint in Jira. The sprint will only be returned if the user can view the board that the sprint was created on, or view at least one of the issues in the sprint.",
+    "instillShortDescription": "Get a sprint in Jira",
+    "input": {
+      "description": "Get an sprint in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "sprint-id"
+      ],
+      "properties": {
+        "sprint-id": {
+          "title": "Sprint ID",
+          "description": "The ID of the sprint. The sprint will only be returned if you can view the board that the sprint was created on, or view at least one of the issues in the sprint.",
+          "instillShortDescription": "The ID of the sprint",
+          "instillUIOrder": 0,
+          "instillFormat": "integer",
+          "instillAcceptFormats": [
+            "integer"
+          ],
+          "instillUpstreamTypes": [
+            "value",
+            "reference",
+            "template"
+          ],
+          "type": "integer"
+        }
+      },
+      "required": [
+        "sprint-id"
+      ],
+      "title": "Input",
+      "type": "object"
+    },
+    "output": {
+      "description": "Get an sprint in Jira",
+      "instillUIOrder": 0,
+      "$ref": "#/$defs/sprint",
+      "required": [],
+      "title": "Output",
+      "type": "object"
+    }
+  },
+  "TASK_CREATE_ISSUE": {
+    "description": "Create an issue in Jira",
+    "instillShortDescription": "Create an issue in Jira",
+    "input": {
+      "description": "Create an issue in Jira",
+      "instillUIOrder": 0,
+      "instillEditOnNodeFields": [
+        "project-key",
+        "issue-type",
+        "status",
+        "summary",
+        "description"
+      ],
+      "properties": {
+        "project-key": {
```

Comment:
question 
how do users get the project-key?
Does issue need to belong to board? Is it a best practice in Jira?

Reply:
I thought the design in Jira is like
![image](https://github.com/user-attachments/assets/3e2bf312-4df9-44f1-a78f-63001e189464)

Reply:
> how do users get the project-key?
All Capital abbreviation behind the name in jira UI is the key, e.g. INS.

> Does issue need to belong to board? Is it a best practice in Jira?
Issues must belong to a project, and, in my understanding, a broad is actually equivalent to a project. Thus, an issue must belong to a board. I choose to follow the description of API document, or should we make it consistent and call all boards as project?